examples/documentation_examples/examples.go

```diff
@@ -2552,7 +2555,11 @@ func CausalConsistencyExamples(client *mongo.Client) error {
 
 	// Use a causally-consistent session to run some operations
 	opts := options.Session().SetDefaultReadConcern(readconcern.Majority()).SetDefaultWriteConcern(
-		writeconcern.New(writeconcern.WMajority(), writeconcern.WTimeout(1000)))
+		&writeconcern.WriteConcern{
```

Comment:
Can we use the convenience function in the example, instead of using the "majority" magic string?

```go
wcMajority := writeconcern.Majority()
wcMajority.WTimeout = 1000
```